dbt_subprojects/tokens/macros/transfers/net_transfers.sql

```diff
@@ -0,0 +1,235 @@
+{% macro evm_net_transfers_daily(blockchain) %} 
+
+with raw_transfers as (
+    select
+        blockchain
+        , block_date
+        , "from" as address
+        , 'sent' as transfer_direction
+        , (sum(amount_usd) * -1) as transfer_amount_usd
+    from
+        {{ source('tokens', 'transfers') }}
```

Comment:
```suggestion
        {{ ref('tokens_transfers') }}
```

Reply:
now that we moved into tokens subproject, we can revert back to using `ref` instead of source, since `tokens_transfers` lives in the same project